QuangF.NV/lession_1/lib/Screen/signScreen/components/body.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:flutter/material.dart';
+import 'package:new_flutter/Screen/signScreen/components/topContent.dart';
+import 'package:new_flutter/Screen/signScreen/components/signInContent.dart';
+import 'package:new_flutter/Screen/signScreen/components/buttonContent.dart';
+import 'package:new_flutter/Screen/signScreen/components/signUp.dart';
+import 'package:new_flutter/constant.dart';
+
+class Body extends StatelessWidget {
+  const Body({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    double height = MediaQuery.of(context).size.height;
+    double _allPadding = 40;
+    double _heightSizedBox = 30;
```

Comment:
:v mấy cái size này e để ra ngoài ý, k để trong build, giả sử có rebuild thì nó lại tạo lại à
```suggestion
class Body extends StatelessWidget {
  const Body({Key? key}) : super(key: key);
  
    final double _allPadding = 40;
    final double _heightSizedBox = 30;
```